examples/logging.rs

```diff
@@ -3,7 +3,7 @@
 use async_std::task;
 
 fn main() {
-    femme::start(log::LevelFilter::Trace).unwrap();
+    femme::with_level(log::LevelFilter::Trace);
```

Comment:
I'd prefer if it comes together with the `femme` update, otherwise, it is in an unbuildable state.